src/manifest.rs

```diff
@@ -28,6 +28,18 @@ quick_error! {
             description("non existent dependency")
             display("The dependency `{}` could not be found in `{}`.", name, table)
         }
+        ParseError(error: String, loline: usize, locol: usize, hiline: usize, hicol: usize) {
+            description("parse error")
+            display("{}:{}{} {}",
```

Comment:
I feel like it's hard to read `format!` strings that use 3 or more anonymous placeholders. Can you give them names? E.g.:

```rust
display("{line}:{col}{upto} {error_msg}",
       line = loline + 1,
       col = locol + 1,
       upto = …)
```